frontend/src/styles/tokens.css

```
@@ -0,0 +1,32 @@
+  --color-code-background: #f4f3ec;
+  --color-accent: #aa3bff;
+  --color-accent-background: rgba(170, 59, 255, 0.1);
+  --color-accent-border: rgba(170, 59, 255, 0.5);
+  --color-chip-border: #3335;
+  --space-chip-inline: 0.6rem;
+  --space-chip-block: 0.1rem;
+  --space-chip-gap: 0.3rem;
+  --space-close-inset: 0.75rem;
+  --radius-chip: 999px;
+  --font-size-close: 1.5rem;
+  --font-family-chip: ui-monospace, Consolas, monospace;
```

Stylelint (17.14.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)